.github/workflows/pr.yml

```diff
@@ -11,16 +11,17 @@ env:
 jobs:
   lint:
     name: Check linting issues
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/cache@v2
+      - uses: actions/checkout@v4
+      - uses: actions/cache@v4
         with:
           path: ~/.npm
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
-      - uses: actions/setup-node@v3
-        with: { node-version: 16, cache: npm }
```

Comment:
Note: removing `node-version` outright should be fine; we just get the latest, when it updates. Back in the days of 10.x/12.x/14.x/16.x, changes were sweeping, and significant. Nowadays the major versions probably won’t break anything (this repo relies on fairly-minimal stuff)

Reply:
![CleanShot 2024-08-16 at 15 51 29@2x](https://github.com/user-attachments/assets/ad75872c-b782-42fd-888d-bb4f5ad993cc)

Whoa using a newer version of Node in GitHub Actions (and newer actions) sped it up by quite a bit! The newer version completed before the old version did (and the old version got a head start, too!)